### DISCUSSION TITLE
Working on the Valve steam deck (Windows)

### DISCUSSION THREAD

**Discussion by ClashSAN** [lshqqytiger/stable-diffusion-webui-directml#14] [General]
I tested 1.5 models at 512x512, speed is 3.6s/it with `--opt-split-attention`

![sd_sd](https://user-images.githubusercontent.com/98228077/221354507-ed82abe7-4b41-4388-91f2-39033acc009d.jpeg)

- `--opt-sub-quad-attention` gave me black squares, and seems slower 
- runs perfect in fp16 (no black squares)
- 7.6s/it for canny 512x512 with controlnet extension with `--opt-split-attention-v1 --medvram`
-  can do larger size: 768x768 and 20s/it, with `--lowvram --opt-split-attention-v1`
- invoke cross attention alone is not enough memory at 512x512

**Comment by oh-pi** [comment]
What bios settings for gpu memory allocation did you use?

**Reply by kroryan** [reply]
how did u installed? when i run ./ webui.sh after few mins my steam deck get reboot

**Reply by ClashSAN** [reply]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/9286

>What bios settings for gpu memory allocation did you use?

4gb

.sh files are for Linux, so we use .bat here.
